l10n_it_fatturapa_out_rc/tests/test_fatturapa_xml_validation.py

```diff
@@ -200,35 +183,36 @@ def test_intra_EU_supplier_refund(self):
         )
 
     def test_extra_EU(self):
-        self.set_sequences(27, "2020-12-01", sequence_name=self.bill_sequence_name)
         self.supplier_extraEU.property_payment_term_id = self.term_15_30.id
+        self.rc_type_eeu.with_supplier_self_invoice = False
```

Comment:
Credo che questa modifica e le successive in questo test facciano cambiare proprio il significato del test, coprendo un caso diverso da quello che copriva in `14.0`.
Come mai sono necessarie per la migrazione?